packages/@react-aria/autocomplete/src/useAutocomplete.ts

```diff
@@ -11,12 +11,12 @@
  */
 
 import {AriaLabelingProps, BaseEvent, DOMProps, RefObject} from '@react-types/shared';
+import {AriaTextFieldProps} from '@react-aria/textfield';
```

Comment:
Can we get this through the `react-aria` monopackage instead of adding a new dependency?

Reply:
wait, do we really want to add `react-aria` as a dependency here? Wouldn't that pull in a lot more?

Reply:
Oh sorry I thought this was in RAC